webapp/lib/view.dart

```diff
@@ -265,21 +265,30 @@ class ChartFiltersView {
   String _periodFilterValue (controller.ChartPeriodFilters filter) {
```

Comment:
The new period filters are not showing up in the dropdown list

Reply:
I've realized that they are only supposed to show on the Driver's tab? My bad

Reply:
no worries - yes, the hour period filters are only for the driver, though it may actually be useful to have them for the other charts too. Let's see how it feels in practice and if they would be useful, we'll add them in.